crmsh/log.py

```diff
@@ -423,6 +400,22 @@ def update_err(self, obj_id, cibadm_opt, xml, rc):
             self.logger.info("offending xml: %s", xml)
 
 
+class ProgressBar:
+    def __init__(self):
+        self._i = 0
+
+    def progress(self):
+        try:
+            width, _ = os.get_terminal_size()
+        except OSError:
+            # not a terminal
+            return
+        self._i = (self._i + 1) % width
+        line = '\r{}>>{}'.format('.' * self._i, '.' * (width - 2 - self._i))
```

Comment:
I think take the whole width of terminal looks too long, maybe this one looks better?
```
INFO: BEGIN Waiting for cluster ...
...........>>
INFO: END Waiting for cluster.
```
This is the code I change:
```
diff --git a/crmsh/bootstrap.py b/crmsh/bootstrap.py
index afe887b0..19488e4a 100644
--- a/crmsh/bootstrap.py
+++ b/crmsh/bootstrap.py
@@ -373,6 +373,7 @@ def wait_for_cluster():
     with logger_utils.status_long("Waiting for cluster") as progress_bar:
         while True:
             if is_online():
+                print()
                 break
             status_progress(progress_bar)
             sleep(2)
diff --git a/crmsh/log.py b/crmsh/log.py
index bee8f6f6..6903616b 100644
--- a/crmsh/log.py
+++ b/crmsh/log.py
@@ -411,7 +411,7 @@ class ProgressBar:
             # not a terminal
             return
         self._i = (self._i + 1) % width
-        line = '\r{}>>{}'.format('.' * self._i, '.' * (width - 2 - self._i))
+        line = '\r{}>>'.format('.' * self._i)
         sys.stdout.write(line)
         sys.stdout.flush()
```
And for sure the caller should remember to print new line:)

Reply:
Make sens to me, I prefer to delete ">>" even.

```
-        line = '\r{}>>{}'.format('.' * self._i, '.' * (width - 2 - self._i))
+        line = '\r{}'.format('.' * self._i)
```

Reply:
> I think take the whole width of terminal looks too long, maybe this one looks better?

It works by consuming the whole width of terminal so that another log written to terminal start from a new line. If it looks too long, `-` can be replaced by space.

Reply:
> replaced by space.
Nice tricky point, to shift the coming text to the new line ;)